department-of-veterans-affairs/caseflow
# JHoang/APPEALS-29213

Resolves [29213](https://jira.devops.va.gov/browse/APPEALS-29213)

# Description
Please explain the changes you made here.

## Acceptance Criteria

## Testing Plan

1. Login as JOLLY_POSTMAN
2. Click on "Your Queue"
3. Click on "Switch Views"
4. Click on "Correspondence Cases"
5. Click on any case
6. On the right, click on the dropdown labeled "Request package action"
7. Click on Remove package from caseflow
8. Check Confirm request button is disabled
9. Fill in the textbox with some text
10. Check Confirm request button is enabled
11. Click Confirm request button and ensure user is taken back to the Correspondence Cases table

# Frontend
## User Facing Changes

 BEFORE|AFTER
 
![Screenshot 2023-12-07 at 11 06 10 AM](https://github.com/department-of-veterans-affairs/caseflow/assets/111081469/db46aecd-4099-4488-81bf-8d43707fffda)

![Screenshot 2023-12-07 at 11 02 36 AM](https://github.com/department-of-veterans-affairs/caseflow/assets/111081469/5af5f8ba-c313-4fc8-8f79-8d1b00a2a00a)


# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
